starter-code/public/scripts/controllers/articleController.js

```diff
@@ -3,10 +3,16 @@ var app = app || {};
 
 (function(module) {
   const articleController = {};
-  // TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
+  // DONE: Setup a function that kicks off the fetching and rendering of articles, using the same
   // code that used to be in index.html.
   // Also be sure to hide all the main section elements, and reveal the #articles section:
-
+  // Estimated 10 min | Actual 20 min
+  articleController.initArticleController = () => {
+    module.Article.fetchAll(function(){
```

Comment:
This is not 100% correct. We want to call exactly what we were calling inside index, on app. So fetchAll with a specific callback that renders the index page. :) We don't want our show and hide inside of fetchAll!